src/redux/reducers/rootReducer.js

```diff
@@ -0,0 +1,55 @@
+import { combineReducers } from 'redux';
+import { authReducer } from './authReducers';
+import { tripReducer } from './tripReducers';
+import * as TripActions from '../actions/tripActions';
```

Comment:
I really like this notation. I've only seen it as
`import TripActions from '../actions/tripActions'`
which seems to function the same way, but this is way more readable

Reply:
This is a cool feature that's new to Javascript es6. I'm not sure if all browsers support it, yet... But I like it wayyy more too.